tests/switchmap_/dashboard/table/test_event.py

```
@@ -0,0 +1,74 @@
+"""Unit tests for the event table functionalities.
+
+This module contains tests for the `table`, `EventTable`, and `EventsRow`
+classes from the `switchmap.dashboard.table.events` module.
+"""
+
+import unittest
+from switchmap.dashboard.table.events import table, EventTable, EventsRow
```

Ruff (0.8.2)

`switchmap.dashboard.table.events.EventsRow` imported but unused

Remove unused import: `switchmap.dashboard.table.events.EventsRow`

(F401)